src/ol_infrastructure/infrastructure/aws/network/__main__.py

```diff
@@ -38,6 +36,19 @@ def vpc_exports(vpc: OLVPC) -> Dict:
         'Name': f'{stack_name} Data Services'})
 data_vpc = OLVPC(data_vpc_config)
 
+residential_mitx_vpc_config = OLVPCConfig(
+    vpc_name=f'mitx-{env_suffix}',
+    cidr_block=get_config('residential_vpc:cidr_block'),
+    num_subnets=4,
```

Comment:
4 not 3?

Reply:
Yeah, it's a quirk of how we manage the subnet ranges between Salt and Pulumi. Salt starts at `x.x.1.0/24` and Pulumi is set to start at `x.x.0.0/24`, so in order to import the 3 existing subnets we need to tell it to manage 4 subnets.